CHANGELOG.md

```diff
@@ -154,6 +159,9 @@
 
 ## Security
 
+
+ [(#2176)]: https://github.com/rust-lang/rust-bindgen/pull/2176
```

Comment:
What are the security implications ? just general "`unsafe` code -> hic sunt dracones" security implications ?

Reply:
Oh, I mostly just was putting the "links" section at the end of that unreleased release section.

Reply:
Oh, right. Rendered, the markdown would not show anything under Security. My mistake.